src/main/java/oshi/hardware/platform/linux/LinuxNetworks.java

```diff
@@ -50,17 +50,19 @@ private long readFile(String file) {
     }
 
     private void setNetworkStats(HWNetworkStore netstore) {
-        String txBytesPath, rxBytesPath, txPacketsPath, rxPacketsPath;
+        String txBytesPath, rxBytesPath, txPacketsPath, rxPacketsPath, speed;
 
         txBytesPath = String.format("/sys/class/net/%s/statistics/tx_bytes", netstore.getName());
         rxBytesPath = String.format("/sys/class/net/%s/statistics/rx_bytes", netstore.getName());
         txPacketsPath = String.format("/sys/class/net/%s/statistics/tx_packets", netstore.getName());
         rxPacketsPath = String.format("/sys/class/net/%s/statistics/rx_packets", netstore.getName());
+        speed = String.format("/sys/class/net/%s/speed", netstore.getName());
```

Comment:
It's ugly, because it isn't a statistic, but I haven't other ideas how to place it